Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -35,6 +35,7 @@ const { check, indexOne } = lgIndexer;
 
```

Comment:
Let me restate my expectation, 

first, you created a completion experience based on a fixed data structure.  That's been done. 
then what we just have to do, is replace this fixed data structure with a delegate than passed in constructor, that's it, no more, no less.